### PR TITLE
More icons and changes

### PR DESCRIPTION
**Pull request by feedmes1eep**
Basically same description as #27 also added the comment as requested in the comments. Changed the template file's line number as well.

### CODE REVIEW

File: BlueStacks/Transparent/BlueStacks Left.ini [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Update=1000
+BackgroundMode=2
+SolidColor=0,0,0,1
+
+[Metadata]
+Name=unFold
+Author=feedmes1eep
+Version= 5.0
+License=GNU General Public License v2.0
+Information=The left side button for BlueStacks
```
Comment (PinkDev1) [inline BlueStacks/Transparent/BlueStacks Left.ini:13]: Interesting to see that you changed the _Information_ section.

I sometimes mess up the _Information_ section and end up writing stuff like:
`Information=Toggl Left Button that is shown on the left side`

File: BlueStacks/Transparent/BlueStacks Left.ini [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+[Rainmeter]
+Update=1000
+BackgroundMode=2
+SolidColor=0,0,0,1
+
+[Metadata]
+Name=unFold
+Author=feedmes1eep
+Version= 5.0
```
Comment (PinkDev1) [inline BlueStacks/Transparent/BlueStacks Left.ini:11]: I don't mind the little space beetwen the _=_ and the version number; But make sure to correct your config file generator for any future PR you make.

**Comment by Ruben35** [comment]
@feedmes1eep well done!

File: BlueStacks/Transparent/BlueStacks Left.ini (hunk 2 above)
Comment (feedmes1eep) [reply BlueStacks/Transparent/BlueStacks Left.ini:11]: Okai!

**Comment by feedmes1eep** [comment]
@PinkDev1 Shall remember for future icons! Thank you for the feedback.

**Pull request merged**